ext/riverlea/Civi/Riverlea/StyleLoader.php

```diff
@@ -121,6 +121,10 @@ public function alterBundles($e): void {
       foreach (self::CORE_FILES as $file) {
         $bundle->addStyleFile('riverlea', "core/css/{$file}");
       }
+      if (\CRM_Utils_Request::retrieve('safe_css', 'Boolean')) {
```

Comment:
Maybe we should only do this if the system is in debug mode (or env is set to dev... I never know which of those 2 settings should control stuff like this, but I'd go 51% for debug mode in this case)